source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -89,6 +89,16 @@ Attributes:
 +                          +------------------------------------------------------------+----------------------------------------------------------+
 |                          | Allowed values                                             | yes, no                                                  |
 +--------------------------+------------------------------------------------------------+----------------------------------------------------------+
+| **diff_size_limit**      | Limit the mazximum size of the file which will report diff information with ``report_changes`` enabled.               |
++                          +                                                                                                                       +
+|                          | Files bigger than this value will not report diff information.                                                        |
++                          +                                                                                                                       +
+|                          | .. versionadded:: 4.0.0                                                                                              |
```

Comment:
Space is missing that causes a warning when generating the table.

Reply:
Done 8cce8d0